unit_tests/service/test_collection_service.py

```diff
@@ -140,31 +139,33 @@ def test_update_raises_when_invalid_id(
 
 # --- CREATE
 
+USER_EMAIL = "test@cpr.org"
+
 
 def test_create(
     collection_repo_mock,
-    organisation_repo_mock,
+    app_user_repo_mock,
 ):
     new_collection = create_dto(import_id="A.0.0.5")
-    collection = collection_service.create(_to_create_dto(new_collection))
+    collection = collection_service.create(_to_create_dto(new_collection), USER_EMAIL)
     assert collection is not None
     assert collection_repo_mock.create.call_count == 1
     # Ensure the collection service uses the org service to validate
-    assert organisation_repo_mock.get_id_from_name.call_count == 1
+    assert app_user_repo_mock.get_org_id.call_count == 1
 
 
 def test_create_when_db_fails(
     collection_repo_mock,
-    organisation_repo_mock,
+    app_user_repo_mock,
 ):
     new_collection = create_dto(import_id="a.b.c.d")
     collection_repo_mock.return_empty = True
     with pytest.raises(RepositoryError) as e:
-        collection_service.create(_to_create_dto(new_collection))
+        collection_service.create(_to_create_dto(new_collection), USER_EMAIL)
     assert e.value.message == "Error when creating collection 'description'"
     assert collection_repo_mock.create.call_count == 1
     # Ensure the collection service uses the geo service to validate
```

Comment:
Cut paste comment needs updating to say collections service uses the app user repo to validate.

Reply:
Oops - removed the check and not the comment